src/options.ts

```diff
@@ -0,0 +1,8 @@
+import { Config } from "./config";
+
+export interface Options {
```

Comment:
@inlined has been working toward a similar goal for the functions deploy codepath: https://github.com/firebase/firebase-tools/blob/master/src/deploy/functions/args.ts#L17. I agree with the approach of trying to standardize these types across the whole codebase, but I think there are some conflicts between these types that should be resolved.

Added him as a reviewer too, since he spent a good bit of time thinking about this recently & may have some insights.

Reply:
Oh nice, I will definitely unify this.